lib/send_sms.rb

```diff
@@ -0,0 +1,26 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class TwilioTest
+
+  def initialize
+    @client = Twilio::REST::Client.new account_sid, auth_token
+  end
+
+  def time
+    time = Time.now.strftime('%H:%M').to_f + 3600
+  end
+
+  def auth
+    account_sid = '***********************'
```

Comment:
Useless assignment to variable - `account_sid`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.